internal/core/util/src/main/java/io/quarkiverse/azure/core/util/AzureQuarkusIdentifier.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkiverse.azure.core.util;
+
+/**
+ * Util class for Azure Quarkus Identifier.
+ *
+ * @implNote The identifier is used to identify the Azure Quarkus Extension, and its max length is limited to 24 characters.
+ * @since 1.0.1
+ */
+public class AzureQuarkusIdentifier {
```

Comment:
How about to make the class final and give it a private constructor.
See [link](https://github.com/Azure/azure-sdk-for-java/blob/2c68397adca8db969f35d5787d5c906213d164bc/sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/util/AzureSpringIdentifier.java#L17C20-L17C41).

Reply:
Done, see [f29bf79](https://github.com/quarkiverse/quarkus-azure-services/pull/142/commits/f29bf7912b3ebd63d335633580278ff393713552).